keyboards/1upkeyboards/sweet16/keymaps/ridingintraffic/keymap.c

```diff
@@ -0,0 +1,293 @@
+/*
+tap danc eis turned on in the rules now...
+*/
+#include "sweet16.h"
+#include "action_layer.h"
+
+#define _EMOJI 0
+#define _TAPLAND 1
+#define _LEDCNTL 2
+
+
+#define TAPPING_TOGGLE 2
+
+enum custom_keycodes {
+    SHRUG,
+    DISFACE,
+    FU,
+    TFLIP,
+    TFLIP2,
+    SAD_EYES,
+    TPUT,
+    HAPPYFACE,
+    HEARTFACE,
+    CLOUD,
+    CHANFACE,
+    CMDCLEAR
+};
+//Tap Dance Declarations
+enum {
+  TD_EXAMPLE1 = 0,
+  TD_EXAMPLE2,
+  TD_EXAMPLE3,
+  TD_EXAMPLE4
+};
+
+void tap(uint16_t keycode){
+    register_code(keycode);
+    unregister_code(keycode);
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    /* EMOJI Pad
+    * ,-------------------------------.
+    * |TFLIP | TFlIP2|ANGRYFL|  TPUT  |
+    * |------+-------+-------+--------|
+    * | CLOUD|       |       | CLEAR  |
+    * |------+-------+-------+--------|
+    * |SHRUG |DISFACE| HRTFAC| HAPPYF |
+    * |------+-------+-------+--------|
+    * | ENTER|       |LEDCNTR| tapland| 
+    * `-------------------------------'
+    */
+      //purple
+    [_EMOJI]{
+       {TFLIP,   TFLIP2,   KC_TRNS,      FU }, 
+       {CLOUD,   KC_TRNS,  KC_TRNS,      CMDCLEAR}, 
+       {SHRUG,   DISFACE,  HEARTFACE,    HAPPYFACE}, 
+       {KC_ENT,  KC_TRNS,  MO(_LEDCNTL), MO(_TAPLAND)}
+    },
+ 
+    /* TapLand
+    * ,-------------------------------.
+    * | str1 | str2  |  str3 |   str4 |
+    * |------+-------+-------+--------|
+    * |      |       |       |        |
+    * |------+-------+-------+--------|
+    * |      |       |       |        |
+    * |------+-------+-------+--------|
+    * |      |       |       |        | 
+    * `-------------------------------'
+    */
+    //blue
+    [_TAPLAND]{
+       {TD(TD_EXAMPLE1), TD(TD_EXAMPLE2),  TD(TD_EXAMPLE3), TD(TD_EXAMPLE4)}, 
+       {KC_TRNS,         KC_TRNS,          KC_TRNS,         KC_TRNS}, 
+       {KC_TRNS,         KC_TRNS,          KC_TRNS,         KC_TRNS}, 
+       {KC_TRNS,         KC_TRNS,          KC_TRNS,         KC_TRNS}
+    },
+    /* LEDControl Pad
+    * ,-------------------------------.
+    * | snake|breathe|rainbow|gradient|
+    * |------+-------+-------+--------|
+    * | xmas |  Val+ | Sat+  |   HUE+ |
+    * |------+-------+-------+--------|
+    * |kngrdr|  Val- | Sat-  |   HUE- |
+    * |------+-------+-------+--------|
+    * | swirl|       | PLAIN | ON/OFF | 
+    * `-------------------------------'
+    */
+    //blue
+    [_LEDCNTL]{
+        {RGB_M_SN, RGB_M_B,    RGB_M_R,     RGB_M_G}, 
+        {RGB_M_X,  RGB_VAI,    RGB_SAI,     RGB_HUI}, 
+        {RGB_M_K,  RGB_VAD,    RGB_SAD,     RGB_HUD}, 
+        {RGB_M_SW, KC_TRNS,    RGB_M_P,     RGB_TOG}
+     },
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    if (record->event.pressed) {
+        switch(keycode) {
+            case CLOUD:       // (っ◕‿◕)っ
+                if(record->event.pressed){
+                    process_unicode((0x28|QK_UNICODE), record);   // Head
```

Comment:
Have you seen this? 
https://docs.qmk.fm/#/feature_unicode?id=send_unicode_hex_string

:)